src/irepl.cpp

```diff
@@ -160,7 +164,7 @@ usage() {
         " ",
         "Options are:",
         " -a  all - if used with -U, update all stale copies",
-        " -B  Backup mode - if a good copy already exists in this",
+        " -B  Backup mode - (Deprecated) if a good copy already exists in this",
```

Comment:
hesitate to suggest this - but consider changing the word 'copy' on this line and the next to be 'replica'... but ... not even sure that is correct/true.

ignore with impunity.

Reply:
I think... I'm going to leave this as-is. Copy might actually be the right word in this case - I simply do not know.

Reply:
Defer.  Resolving.